performancetest/src/main/java/org/corfudb/performancetest/CorfuTablePerformanceTest.java

```diff
@@ -0,0 +1,193 @@
+                try {
+                    putTable(table, random.nextInt(randomBoundary));
+                    getTable(table, random.nextInt(randomBoundary));
+                } catch (Exception e) {
+                    log.error("Operation failed with", e);
+                }
+            }
+        });
+    }
+
+    /**
+     * Performance test for using 1 CorfuTable and 1 Thread doing putTable and getTable operations.
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    @Test
+    public void CorfuTable1Thread1TableTest () throws IOException, InterruptedException {
+        setMetricsReportFlags("corfutable-1-1");
+        Process server = runServer();
+        runtime = initRuntime();
+        CorfuTable<String, String>
+                corfuTable = buildTable("table1");
+        long start = System.currentTimeMillis();
+        while (true) {
+            if ((System.currentTimeMillis() - start) / MILLI_TO_SECOND > time) {
+                break;
+            }
+            putTable(corfuTable, random.nextInt(randomBoundary));
+            getTable(corfuTable, random.nextInt(randomBoundary));
+        }
+        killServer(server);
+    }
+
+    /**
+     * This funtion create given number tables and given number threads and
+     * start the Thread to do putTable and getTable operations.
+     * @param numThreads Number of threads.
+     * @param numTables Number of tables.
+     * @throws InterruptedException
+     */
+    private void runTableOps(int numThreads, int numTables) throws InterruptedException {
+        CorfuTable<String, String>[] corfuTables = new CorfuTable[numTables];
+        for (int i = 0; i < numTables; i++) {
+            corfuTables[i] = buildTable("table" + i);
+        }
+        Thread[] threads = new Thread[numThreads];
+        long start = System.currentTimeMillis();
+        for (int i = 0; i < numThreads; i++) {
+            threads[i] = createThread(corfuTables[i], start);
+            threads[i].start();
+        }
+        for (int i = 0; i < numThreads; i++) {
+            threads[i].join();
+        }
+    }
+
+    /**
+     * Performance test for having 10 threads doing putTable and getTable operations for
+     * a single CorfuTable.
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    @Test
+    public void CorfuTable10Thread1TableTest () throws IOException, InterruptedException {
+        setMetricsReportFlags("corfutable-10-1");
+        Process server = runServer();
+        runtime = initRuntime();
+        runTableOps(1, 10);
+        killServer(server);
+    }
+
+    /**
+     * Performance test for having 10 threads doing putTable and getTable operations for
+     * 10 CorfuTables. In this test, threads don't share tables.
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    @Test
+    public void CorfuTable10Thread10Table() throws IOException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)